src/plays/cricket-game/Readme.md

```diff
@@ -0,0 +1,29 @@
+# Cricket Game
+
+Play Cricket and Bat against the computer to chase down the given target of runs with few overs and wickets in hand. You will level up if you successfully chase the target or else you will lose! Hit the ball carefully when it comes to you! 🏏
+
+## Play Demographic
+
+- Language: js
+- Level: Intermediate
+
+## Creator Information
+
+- User: SamirMishra27
+- Gihub Link: https://github.com/SamirMishra27
+- Blog: 
+- Video: 
```

Comment:
Have you recorded the video stream?

Reply:
I have recorded a video of me demonstrating the project in motion. 
I was unable to record or stream myself coding this project live.

Reply:
Demo recording has been added in the `Readme.md` file of react-play.
https://github.com/reactplay/react-play/pull/972/commits/c7dba9eedf67f7176c2f523e63b23b6d6720673e